src/main/java/org/concordion/api/AbstractCommand.java

```diff
@@ -25,4 +25,9 @@ public boolean isExample() {
 
     public void executeAsExample(CommandCall commandCall, Evaluator evaluator, ResultRecorder resultRecorder) {
     }
+
+    public boolean alwaysRunEvenIfNoNonExampleChildren() {
+        return false;
+    }
+

```

Comment:
Or could we just use the existing `isExample()` method? This would match `ExampleCommand` as well as `TableRowExampleCommand` so might not be what is needed.